data/com.usebottles.bottles.metainfo.xml.in

```diff
@@ -6,9 +6,9 @@
   <launchable type="desktop-id">com.usebottles.bottles.desktop</launchable>
   <name>Bottles</name>
   <summary>Run Windows Software</summary>
-  <developer_name translatable="no">Bottles Contributors</developer_name>
+  <developer_name translate="no">Bottles Contributors</developer_name>
   <developer id="com.usebottles">
-    <name translatable="no">Bottles Contributors</name>
+    <name translate="no">Bottles Contributors</name>
```

Comment:
Should `Contributors` be translatable?

Reply:
If it was marked as not translatable before, there might be a reason. Would need more discussion. Let's keep as-is for now.